src/client/Microsoft.Identity.Client/Cache/Items/MsalAccessTokenCacheItem.cs

```diff
@@ -28,7 +30,8 @@ internal MsalAccessTokenCacheItem(
             string tenantId,
             string homeAccountId,
             string keyId = null,
-            string oboCacheKey = null)
+            string oboCacheKey = null,
+            IEnumerable<string> additionalRequestRapameters = null)

```

Comment:
name suggestion: `additionalRequestResponseValues`

Reply:
What about `RequestedResponseParameters`. as in, these are the response parameters requested by the dev

Reply:
How about `PersistedResponseParameters` ?